recipes/qt/5.x.x/conanfile.py

```diff
@@ -151,7 +151,7 @@ def validate_build(self):
             # In any case, check its actual version for compatibility
             from six import StringIO  # Python 2 and 3 compatible
             mybuf = StringIO()
-            cmd_v = f"\"{python_exe}\" --version"
+            cmd_v = f"\"{python_exe}\" --version 2>&1"
```

Comment:
This won't work on Windows - You might need to use `subprocess` to execute the command directly and get stderr from there

I'll look into allowing Conan's run method to also output stderr just like what is done with stdout

Reply:
Hi Ruben,

Thanks for reviewing.
Before submitting I tested the "`python --version 2>&1`" command in a Windows command prompt and it worked.
A colleague of mine is currently building qt 5.15.11 on Windows and with the above patched he has no errors yet. Before the patch the build would quit almost immediately because of the broken Python2 version check.
So I am quite confident this solution will work also on Windows. 

Of course it would still be nice if stderr could be re-directed as well.

Reply:
@wdobbe Please, let's avoid workarounds and follow Ruben's approach, the `subprocess` is specially designed for this case, it's cross-platform compatibly, has fine-grained control over pipes, is more secure (avoids injections), and much better for error handling in case of an exception.

Reply:
Changed as requested and tested with Conan 2.13.0 on Linux x86_64 with:
`conan create . --version=5.15.11 -o qt/*:qtwebengine=True -o sqlite3/*:shared=True -o qt/*:shared=True -o qt/*:gui=True -o qt/*:qtdeclarative=True -o qt/*:qtlocation=True -o qt/*:qtwebchannel=True --build=missing`